readme.md

```diff
@@ -87,6 +87,17 @@ function extraSlowJob (cb) {
 extraSlowJob.timeout = 500
 q.push(extraSlowJob)
 
+// jobs can also opt-out of the timeout altogether
```

Comment:
see https://github.com/jessetane/queue/pull/64#issuecomment-500495720

Reply:
i see your point but a 10s timeout in the example is probably a little cumbersome, maybe we can shorten it a little?

Reply:
Done!

Reply:
nice - did you see the comment i linked about the readme and example being the same?
https://github.com/jessetane/queue/pull/64#issuecomment-500495720

Reply:
👆@joelgriffith

Reply:
Sorry, somehow I must have missed this one, added to the example (which runs)